<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowTitle">
   <string>Audio Control</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <!-- <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property> -->
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <!-- <property name="styleSheet">
       <string notr="true">
        background: transparent;
        border: none;
       </string>
      </property> -->
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="sessionContainer">
       <!-- <property name="styleSheet">
        <string notr="true">background: transparent;</string>
       </property> -->
       <layout class="QVBoxLayout" name="sessionLayout"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
